python/tests/res/job_queue/test_job_dispatch.py

```diff
@@ -0,0 +1,96 @@
+from tests import ResTest
+from res.enkf import ResConfig
+from ecl.util.test import TestAreaContext
+
+import os, stat
+import json
+from subprocess import Popen
+import time
+import psutil
+
+class JobDispatchTest(ResTest):
+
+    def setUp(self):
+        self.case_directory = self.createTestPath("local/poly_normal/")
+
+    def testTerminateJobs(self):
+
+    
+
+        with TestAreaContext("job_dispatch_test") as work_area:
+            work_area.copy_directory(self.case_directory)
+
+            #executes it self recursively and sleeps for 100 seconds
+            with open("dummy_executable", "w") as f:
+                f.write("#!/usr/bin/env python\n"\
+                        "import sys, os, time\n"\
+                        "counter = eval(sys.argv[1])\n"\
+                        "if counter > 0:\n"\
+                        "    os.fork()\n"\
+                        "    os.execv(sys.argv[0],[sys.argv[0], str(counter - 1) ])\n"\
+                        "else:\n"\
+                        "    time.sleep(100)\n"
+                    )
+                    
+            executable = os.path.realpath("dummy_executable")
+            os.chmod("dummy_executable", stat.S_IRWXU |  stat.S_IRWXO |  stat.S_IRWXG )
+            
+            self.job_list = {
+                "umask" : "0002",
+                "DATA_ROOT": "",
+                "global_environment" : {},
+                "global_update_path" : {},
+                "jobList" : [ {
+                    "name" : "dummy_executable",
+                    "executable" : executable,
+                    "target_file" : None,
+                    "error_file" : None,
+                    "start_file" : None,
+                    "stdout" : "dummy.stdout",
+                    "stderr" : "dummy.stderr",
+                    "stdin" : None,
+                    "argList" : ['3'],
+                    "environment" : None,
+                    "exec_env" : None,
+                    "license_path" : None,
+                    "max_running_minutes" : None,
+                    "max_running" : None,
+                    "min_arg" : 1,
+                    "arg_types" : [],
+                    "max_arg" : None
+                }],
+                "ert_version" : [],
+                "run_id" : "",
+                "ert_pid" : ""
+                }
+
+            rc = ResConfig("poly_normal/poly.ert")
+            job_dispatch_script = os.path.realpath(rc.queue_config.job_script)
+            
+            with open("jobs.json", "w") as f:
+                f.write(json.dumps(self.job_list))
+
+            
+            #Required to execute job_dispatch in separate process group
+            with open("job_dispatch_executer",'w') as f:
+                f.write("#!/usr/bin/env python\n"\
+                        "import os, sys\n"\
+                        "os.setsid()\n"
```

Comment:
add a coment to this line saying that this si what makes it live in a separate group